src/D2L.Security.OAuth2/Keys/Caching/InMemoryPublicKeyCache.net50.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using D2L.Security.OAuth2.Keys.Default;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace D2L.Security.OAuth2.Keys.Caching {
+	internal sealed partial class InMemoryPublicKeyCache : IInMemoryPublicKeyCache {

```

Comment:
New stuff using the `Microsoft.Extensions.Caching.Memory` NuGet package, as recommended by https://docs.microsoft.com/en-us/aspnet/core/performance/caching/memory?view=aspnetcore-5.0

Reply:
We could make the old stuff use this too, but that could be it's own PR after some investigation.